server/service/src/auth.rs

```diff
@@ -293,13 +292,6 @@ fn all_permissions() -> HashMap<Resource, PermissionDSL> {
             PermissionDSL::HasPermission(Permission::PatientMutate),
         ]),
     );
-    map.insert(
```

Comment:
Deliberately removing permissions?

Reply:
Ah I see you are using the MutateProgram permission. That seems like a good idea.

Reply:
Yes I renamed the endpoint and MutateProgram seemed more appropriated.